drivers/docker/config.go

```diff
@@ -258,6 +258,13 @@ var (
 			hclspec.NewLiteral(`"gcr.io/google_containers/pause-amd64:3.0"`),
 		),
 
+		// the duration that the driver will wait for activity from the Docker engine during an image pull
+		// before canceling the request
+		"pull_activity_timeout": hclspec.NewDefault(
+			hclspec.NewAttr("pull_activity_timeout", "string", false),
+			hclspec.NewLiteral(`"2m"`),
```

Comment:
is "activity" just any feedback on progress, or "docker pull need to complete within this time"?

Reply:
Just any feedback on progress. I've noticed that with Windows images, sometimes the Docker engine will become busy for >2m when extracting a large-ish (~1GB) layers, and Nomad times out the pull.